services/mgmt/applicationinsights/src/package_2020_04/operations.rs

```diff
@@ -5541,7 +5541,7 @@ pub mod my_workbooks {
                         .await
                         .map_err(Error::GetToken)?;
                     req_builder = req_builder.header(http::header::AUTHORIZATION, format!("Bearer {}", token_response.token.secret()));
-                    url.query_pairs_mut().append_pair("api-version", super::API_VERSION);
+                    url.query_pairs_mut().append_pair("api-version", "2015-05-01");
```

Comment:
In this example, the applicationinsights package_2020_04 is using multiple APIs . Here it uses one.